synapse_rock/rockcraft.yaml

```diff
@@ -99,6 +99,10 @@ parts:
             cp pyproject.toml poetry.lock /synapse/
             /usr/local/bin/poetry export --extras all -o /synapse/requirements.txt
             pip3 install --prefix="/install" --no-deps --no-warn-script-location -r /synapse/requirements.txt
+            git clone https://github.com/amandahla/synapse_stats_exporter.git
```

Comment:
If it's not a snap anymore, shouldn't we make it part of this repository ? (Or create a repo under the canonical group at least ?)

Reply:
Good point, I transferred the ownership now.